vignettes/our_plots.Rmd

```diff
@@ -91,8 +91,8 @@ The function will return an error message if there is a typo in the analyte name
 plot_mfi_for_analyte(plate, "Spike_B16172")
 ```
 
-This plot shows the distribution of the MFI values for test samples for the analyte "OC43_NP_NA". The test samples are colored in blue, and the standard curve samples are colored in red.
-The default plot type is violin, but there is an option to change it to violin plot by setting the `plot_type` parameter to `boxplot`.
+This plot shows the distribution of the MFI values for test samples for the analyte "OC43_NP_NA". The test samples are coloured in blue, and the standard curve samples are coloured in red.
+The default plot type is violin, but the option is to change it to violin plot by setting the `plot_type` parameter to `boxplot`.
```

Comment:
Here, both versions are incorrect 

"The default plot type is violin, but there is an option to change it to boxplot by setting the `plot_type` parameter to `boxplot`." 
is correct

Reply:
changed to 
```

This plot shows the distribution of the MFI values for test samples for the analyte "OC43_NP_NA". The test samples are coloured in blue, and the standard curve samples are coloured in red.
The default plot type is violin, but there is an option to change it to the boxplot by setting the `plot_type` parameter to `boxplot`.
```